docs/modules/install/pages/index.adoc

```diff
@@ -120,27 +120,26 @@ You can configure it with `crontab -e`, for instance if you've created your Deci
 
 === Further configuration
 
-We also have other guides on how to configure some extra components:
+We also have other guides on how to configure some extra mandatory components:
 
 * xref:services:activejob.adoc[ActiveJob]: For background jobs (like sending emails).
-* xref:services:maps.adoc[Maps]: How to enable geocoding for proposals and meetings.
```

Comment:
Do we want to remove the reference for the maps services?

It is true that the services provided by the core can be now configured with ENV vars but I believe any custom providers still cannot.

Reply:
No, we don't. I think I missed that one, nice catch!